CHANGELOG.md

```diff
@@ -89,6 +89,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#3524](https://github.com/open-telemetry/opentelemetry-python/pull/3524))
 - Handle `taskName` `logrecord` attribute
   ([#3557](https://github.com/open-telemetry/opentelemetry-python/pull/3557))
+- Implement client key file and client certificate file for all OTLP exporters
```

Comment:
This needs to be moved to unreleased above

Reply:
Hi @xrmx , I am continuing the fix in https://github.com/open-telemetry/opentelemetry-python/pull/4116, could you help take a look?